### DISCUSSION TITLE
there should be up to date help guides on where and how to install any new version of sunshine on the main page for any device.

### DISCUSSION THREAD

**Discussion by marv-the-gamer** [LizardByte/community#473] [Sunshine] [answered]
I have no clue on how to or where to download for the most recent version of sunshine because YouTube does not give me the exact version I'm searching for because I only get outdated tutorials that are no help. Putting updated tutorials on proper installation for anything on this website is important. I put this as general because I have no clue what section this goes in.

**Comment by ReenigneArcher** [comment, marked as answer]
Everything is clearly documented. You can start with the readme and go to the full docs from there.

https://github.com/LizardByte/Sunshine?tab=readme-ov-file#overview
